UQ-eLIPSE/crucible-components
# story/VETHUB-26

### I should not be able to interact with the options after submission

To test:

- yarn run dev
- Navigate to http://localhost:5173/
- Select an option by clicking on it and submit.

Verify:

- Interaction with the options after submission is no longer available.

Run tests:

- yarn run test

### I should be able to select and deselect an option

To test:

- yarn run dev
- Navigate to http://localhost:5173/
- Select an option by clicking on it.
- Click again on the same option.

Verify:

- Clicked option is not selected anymore.
- Submit button is disabled.

Run tests:

- yarn run test
